source/linux/Foundational_Components/Power_Management/pm_low_power_modes.rst

```diff
@@ -106,8 +107,6 @@ Deep Sleep mode using one of the supported wakeup sources.
 MCU Only
 ********
 
-.. _pm_mcu_only:
-
```

Comment:
`pm_wakeup_sources.rst` still contains a ref to this tag.

Reply:
Good catch @StaticRocket !

Reply:
Updated